.github/workflows/android-build-test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test android build
+on:
+  pull_request:
+    branches: [master]
+jobs:
+  test:
+    runs-on: macos-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - name: Install dependencies
+        run: yarn && cd TestsExample && yarn
+      - name: run tests
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: 29
```

Comment:
I am not sure about this one. We support Android versions back to `16` API level, so maybe we should test against the lowest available version? wdyt @kacperkapusciak ?

Reply:
I just copy pasted this value. I'll change it to 16

Reply:
I also changed `minSdkVersion` of TestsExample to 16 (previously 21)

Reply:
I changed it to 21